### PR TITLE
Forbid global window reference

### PR DESCRIPTION
**Pull request by jridgewell**
`window` variable was still leaking into the service worker thorough the `urls` config. `self` is equivalent to `window` (`self === window`), and is safely defined in both the Browser context and the ServiceWorker.

### CODE REVIEW

**Comment by jridgewell** [comment]
PTAL.

**Comment by erwinmombay** [comment]
@jridgewell do we need to also ban assignment to window.self (and implicit)?

**Comment by jridgewell** [comment]
> do we need to also ban assignment to window.self (and implicit)?

Nah, I think this is fine for now.

**Comment by erwinmombay** [comment]
@jridgewell LGTM

File: src/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-global": 2,
```
Comment (erwinmombay) [inline src/.eslintrc]: remove trailing ,

**Pull request merged**